OpenRA.Game/Server/Server.cs

```diff
@@ -121,10 +121,13 @@ public void EndGame()
 				t.GameEnded(this);
 		}
 
-		public Server(IPEndPoint endpoint, ServerSettings settings, ModData modData, bool dedicated)
+		public Server(IPAddress address, int port, ServerSettings settings, ModData modData, bool dedicated)
 		{
 			Log.AddChannel("server", "server.log");
 
+			if (port < 49152 | port > 65534)
+				throw new System.Net.Sockets.SocketException(10022);
```

Comment:
I'm all for hard-blocking ports outside this range from the in-game UI, but somewhere near 100% of our community hosted dedicated servers will also be caught by this.  Considering that we rely on the good-will of these hosts, forcing this on dedicated servers seems like a really bad idea.

Can we please relax this to a warning in the server log?